powerauth-push-server/src/main/java/com/wultra/push/controller/rest/InboxController.java

```diff
@@ -74,7 +76,11 @@ public InboxController(InboxService inboxService) {
     @Operation(summary = "Post a message to inbox", description = "Post a message to inbox to be delivered to users")
     public ObjectResponse<GetInboxMessageDetailResponse> postMessage(
             @Valid @RequestBody ObjectRequest<CreateInboxMessageRequest> request) throws AppNotFoundException {
-        return new ObjectResponse<>(inboxService.postMessage(request.getRequestObject()));
+        final CreateInboxMessageRequest requestObject = request.getRequestObject();
+        final GetInboxMessageDetailResponse response = inboxService.postMessage(requestObject);
+        logger.info("action: postMessage, state: succeeded, applicationId: {}, messageId: {}", 
```

Comment:
the applicationId and messageId should be logged in the entry log, not exit log.